build.gradle.kts

```diff
@@ -27,10 +27,10 @@ dependencies {
     // error-prone-contrib only has provided dependencies, whereas the platform needs these on the classpath at runtime
     runtimeOnly("org.junit.jupiter:junit-jupiter-api:latest.release")
     runtimeOnly("org.assertj:assertj-core:latest.release")
-    runtimeOnly("org.springframework:spring-context:6.0.23")
-    runtimeOnly("org.springframework:spring-test:6.0.23")
-    runtimeOnly("org.springframework:spring-web:6.0.23")
-    runtimeOnly("org.springframework:spring-webflux:6.0.23")
+    runtimeOnly("org.springframework:spring-context:5.3.32")
+    runtimeOnly("org.springframework:spring-test:5.3.32")
+    runtimeOnly("org.springframework:spring-web:5.3.32")
+    runtimeOnly("org.springframework:spring-webflux:5.3.32")
```

Comment:
We might want to adopt what the folks Picnic use themselves:
https://github.com/PicnicSupermarket/error-prone-support/blob/078d8c16fa4145492eec2099c3f3cb55d2987e38/pom.xml#L500-L518